ord_schema/validations_test.py

```diff
@@ -175,6 +175,19 @@ def test_reaction(self):
                                     'reaction input'):
             self._run_validation(message)
 
+    def test_reaction_smiles(self):
+        message = reaction_pb2.Reaction()
+        message.identifiers.add(value='test', type='REACTION_SMILES')
+        output = self._run_validation(message)
+        self.assertEmpty(output.errors)
+        self.assertEmpty(output.warnings)
```

Comment:
If adding a Reaction SMILES value check, this warning list will not be empty

Reply:
yes, i'll add this check